### PR TITLE
Print out percentage

### PR DESCRIPTION
**Pull request by YafeiHu**
https://trello.com/c/nFwccyfx/43-logging-batch-completion-check

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ const BUTTON_SELECTOR = "button[type=submit]";
 const CAQH_INPUT_SELECTOR = `#CAQHProviderId`;
 const DEFAULT_DELAY_INTERVAL = 5000;
 const TIMEOUT = 90000;
+const BATCH_SIZE = 100;
```
Comment (mike-singh) [inline index.js]: You will need to make a small modification here, if a row number is passed in, we need to figure out the difference between that row number and 100.

Comment (YafeiHu) [reply index.js]: @mike-singh I don't understand, To me `${Math.floor(successCount/BATCH_SIZE*100)}% complete` is already handling this?

Comment (mike-singh) [reply index.js]: https://github.com/CredSimple/exorcist/pull/40/files

you can get less than 100 rows

Comment (mike-singh) [reply index.js]: You can probably do ```csvRows.length || 100```

Comment (YafeiHu) [reply index.js]: @mike-singh I don't think it will work, since we don't have csvRows here. But I can do a similar fix.

**Pull request merged**